edweiss-app/__test__/app/(app)/lectures/remotecontrol/index.test.tsx

```diff
@@ -23,7 +26,13 @@ jest.mock('@/config/i18config', () => ({
 }));
 
 jest.mock('expo-router', () => ({
-    useLocalSearchParams: jest.fn(),
+    router: { push: jest.fn() },
+    Stack: {
+        Screen: jest.fn(({ options }) => (
+            <>{options.title}</> // Simulate rendering the title for the test
+        )),
+    },
+    useLocalSearchParams: jest.fn(() => ({ courseNameString: 'testCourse', lectureIdString: 'testLectureId' }))
```

Comment:
Good catch 😋, yes I agree with your mock. It makes more sens to directly put the `useLocalSearchParam` within the `expo-router` mock.